src/PowerSystems2PRAS.jl

```diff
@@ -881,6 +1045,9 @@ function generate_pras_system(
 end
 
 const DEFAULT_DEVICE_MODELS = [
+    DeviceRAModel(PSY.Line, LinePRAS),
+    DeviceRAModel(PSY.MonitoredLine, LinePRAS),
+    DeviceRAModel(PSY.TwoTerminalHVDCLine, LinePRAS),
```

Comment:
Should we just make AreaInterchange part of the default device model as well? I think that is the right thing to use to populate PRAS Interfaces if they exist.